### ISSUE TITLE
Global config of query string tracing fails with `AttributeError`

### ISSUE THREAD

**Issue by jprestel-rue** [DataDog/dd-trace-py#1573]
### Which version of dd-trace-py are you using?
`0.40.0`

### Which version of the libraries are you using?
on python 3.6.10

can provide library list if really necessary

### How can we reproduce your problem?
As per the instructions in [advanced usage](https://github.com/DataDog/dd-trace-py/blob/bffcaf3c612db3c6064c936a37e932d3293c878a/docs/advanced_usage.rst#query-string-tracing):

```python
from ddtrace import config

# Global config
config.http.trace_query_string = True
```

### What is the result that you get?
This fails with `AttributeError: can't set attribute`

```python
>>> from ddtrace import config
>>> config.http.trace_query_string = True
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/opt/storefrontpy/lib/python3.6/site-packages/ddtrace/utils/attrdict.py", line 34, in __setattr__
    object.__setattr__(self, key, value)
AttributeError: can't set attribute
>>>
```

### What is the result that you expected?
Should successfully configure this setting globally

**Comment by Kyle-Verhoog** [comment]
Yep, can confirm that this has been broken since it was introduced in #886 😕.

**Comment by sacharya1980** [comment]
Are there any known workarounds to enable query string tracing? I just ran into this today, and it's blocking me 🙁

**Comment by Kyle-Verhoog** [comment]
@sacharya1980 yes, you can enable it on an integration-by-integration basis. So if you're using django it would be `ddtrace.config.django.trace_query_string=True`. We're going to have a fix for it in v0.42.
